### ISSUE TITLE
certificate verify failed

### ISSUE THREAD

**Issue by rezso** [regosen/gallery_get#3]
Unfortunately, another issue.
This is related to python 2.7.9 (https://www.python.org/dev/peps/pep-0476/):
------------------------------------------------------------
Traceback (most recent call last):
  File "gallery_get.py", line 231, in run
    self.run_internal()
  File "gallery_get.py", line 203, in run_internal
    rpage = urllib.urlopen(info.redirect).read()
  File "/usr/lib/python2.7/urllib.py", line 87, in urlopen
    return opener.open(url)
  File "/usr/lib/python2.7/urllib.py", line 213, in open
    return getattr(self, name)(url)
  File "/usr/lib/python2.7/urllib.py", line 364, in open_http
    return self.http_error(url, fp, errcode, errmsg, headers)
  File "/usr/lib/python2.7/urllib.py", line 377, in http_error
    result = method(url, fp, errcode, errmsg, headers)
  File "/usr/lib/python2.7/urllib.py", line 641, in http_error_302
    data)
  File "/usr/lib/python2.7/urllib.py", line 667, in redirect_internal
    return self.open(newurl)
  File "/usr/lib/python2.7/urllib.py", line 213, in open
    return getattr(self, name)(url)
  File "/usr/lib/python2.7/urllib.py", line 443, in open_https
    h.endheaders(data)
  File "/usr/lib/python2.7/httplib.py", line 997, in endheaders
    self._send_output(message_body)
  File "/usr/lib/python2.7/httplib.py", line 850, in _send_output
    self.send(msg)
  File "/usr/lib/python2.7/httplib.py", line 812, in send
    self.connect()
  File "/usr/lib/python2.7/httplib.py", line 1212, in connect
    server_hostname=server_hostname)
  File "/usr/lib/python2.7/ssl.py", line 350, in wrap_socket
    _context=self)
  File "/usr/lib/python2.7/ssl.py", line 566, in __init__
    self.do_handshake()
  File "/usr/lib/python2.7/ssl.py", line 788, in do_handshake
    self._sslobj.do_handshake()
IOError: [Errno socket error] [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:581)
Using params: ['http://imagearn.com/gallery.php?id=592017', '/home/rezso/bin/get', False]
------------------------------------------------------------

**Comment by regosen** [comment]
Ok, try grabbing latest.  There is no specific plugin for imagarn galleries, so it falls back on the generic plugin.  I found that the generic plugin works well for imagearn, except some of the non-image redirects were causing errors.  I updated the script to account for this.

BTW- Since it's using a generic plugin, it will probably download a few extra graphics images from imagearn's website- just delete them afterward.  This is expected when using the generic plugin.
